ci/actions-templates/windows-builds-template.yaml

```diff
@@ -113,6 +113,12 @@ jobs:
       # A script might be nicer, though these in theory could be made to run
       # independently, assuming the run above succeeded, allowing us to run
       # the full test suite rather than stop on first error, at least in PRs
+      - name: Run the CARGO_HOME in self-installation step
+        run: |
+          set -e
+          rm ~/.cargo/bin/rustup
+          target/${{ matrix.target }}/release/rustup-init.exe -y --default-host ${{ matrix.target }} --default-toolchain=stable --profile=minimal
+        shell: bash
```

Comment:
It's not clear to me what your goal is here.  Why are you installing the built rustup into the test host at this time?
Given this is part of a `do not merge` commit, did you mean to leave it in?

Reply:
I don't have a Windows machine and I don't wanna run the virtual machine.
So to check if the new installation prompt for Windows displays correctly, 
I add this commit.